includes/multi-currency/AdminNotices.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Admin notices for Multi-Currency.
+ *
+ * @package WooCommerce\Payments\MultiCurrency
+ */
+
+namespace WCPay\MultiCurrency;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class that will display admin notices.
+ */
+class AdminNotices {
+	/**
+	 * Notices.
+	 *
+	 * @var array
+	 */
+	private $notices = [];
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		add_action( 'admin_notices', [ $this, 'admin_notices' ] );
+		add_action( 'wp_loaded', [ $this, 'hide_notices' ] );
+	}
+
+	/**
+	 * Display any notices we've collected thus far.
+	 */
+	public function admin_notices() {
+		if ( ! current_user_can( 'manage_woocommerce' ) ) {
+			return;
+		}
+
+		$this->check_for_notices();
+
+		foreach ( (array) $this->notices as $notice_key => $notice ) {
```

Comment:
Do we need to cast to an array with `(array)` here? `$this->notices` is set to an empty array when the class is initialized, and the only other place it gets changed is in the `add_admin_notices` function where we add elements to the array, so we should be able to rely on this always being an array.

Reply:
Good catch :) 
This was a stripped down copy of the `Admin_Notices` class, and that casting was in there, so it's my fault for not removing it. Removed in https://github.com/Automattic/woocommerce-payments/pull/2322/commits/b6ccb7d5165686c2bc4fdee0932a24f949d8fdfa

Reply:
Thanks for updating! And no worries, It wouldn't have caused any issues with the code, just something that caught my eye when reviewing. :)